src/Bread.sol

```diff
@@ -89,6 +93,12 @@ contract Bread is
 
         emit ClaimedYield(amount);
     }
+    function claimYieldForDisbursement() external {
```

Comment:
Wondering if having "both" of these yield claimers is problematic ? Should we only keep this new one ?

Reply:
added only an owner modifier to it, I think it's fine for us to have it in case of errors for now. later on should be removed.